.ci/travis.sh

```diff
@@ -131,7 +131,10 @@ jacoco)
     jacoco:check@default-check
   # BUILD_REASON is variable from CI, if launch is not from CI, we skip this step
   if [ -n "$BUILD_REASON" ];then
+    echo "Reporting to codecov"
```

Comment:
> No reporting to codecov outside CI

Variable is not working. I am going to try sevntu's version of `if [[ $TRAVIS == 'true' ]]; then`.